.jenkins/.pipeline/lib/config.js

```diff
@@ -6,7 +6,7 @@ const name = 'jenkins';
 
 const phases = {
```

Comment:
why do you still need the pipeline setup for jenkins?

Reply:
this is for the jenkins image itself. I can't remove this until all pipeline scripts have been migrated

Reply:
which won't happen since we also manage reggie and stuff